Makefile

```diff
@@ -307,12 +307,39 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)_$(APPLICATION_VERSION_DEBIAN)_$(
 		--extraMetadata.packageType=deb \
 		$(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage: \
-	| $(BUILD_DIRECTORY)
-	$(NPX) build --linux AppImage $(ELECTRON_BUILDER_OPTIONS) \
+ifeq ($(TARGET_ARCH),x64)
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-unpacked
+else
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-$(TARGET_ARCH_ELECTRON_BUILDER)-unpacked
+endif
+
+$(BUILD_DIRECTORY)/$(ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY):
+	$(NPX) build --dir --linux $(ELECTRON_BUILDER_OPTIONS) \
+		--extraMetadata.name=$(APPLICATION_NAME_ELECTRON) \
 		--extraMetadata.version=$(APPLICATION_VERSION) \
 		--extraMetadata.packageType=AppImage
 
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(PLATFORM).AppDir: \
```

Comment:
To keep things consistent, should this line (and line 335) be using `$(APPLICATION_NAME_LOWERCASE)` instead of `$(APPLICATION_NAME)` ?